docs/user_manual/source/rvfi.rst

```diff
@@ -165,10 +165,18 @@ Instead of:
    output [31:0] rvfi_csr_name31_wdata
 
 
+**Machine Counter/Timers**
+
+In contrast to [SYMBIOTIC-RVFI]_, the **mcycle** and **minstret** registers are not modelled as happening "between instruction" but rather as a side-effect of the instruction.
```

Comment:
**mcycle** and **minstret** -> **mcycle[h]** and **minstret[h]**

Reply:
Pushed a fix